.github/workflows/pr-test-e2e.yml

```diff
@@ -316,4 +323,4 @@ jobs:
           kubectl --context cluster1 port-forward -n open-cluster-management-agent-addon deploy/status-agent 8080 &
           sleep 10
           curl http://localhost:8080/metrics
-          kill %
+          kill %
```

Comment:
Not loving the red ink here.

<img width="160" height="94" alt="Image" src="https://github.com/user-attachments/assets/758e8fe2-fe72-4789-bb7d-21858a88887b" />

Reply:
It's baaack.